docs/source.go

```diff
@@ -0,0 +1,106 @@
+// Package docs helps create plugin documentation
+package docs
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/cloudquery/plugin-sdk/plugins"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+)
+
+const (
+	tablesDir = "tables"
+)
+
+// GenerateSourcePluginDocs creates table documentation for the source plugin based on its list of tables
+func GenerateSourcePluginDocs(p *plugins.SourcePlugin, outputPath string, deleteOld bool) error {
+	if deleteOld {
```

Comment:
I think we should never delete files and it should be part of someone script if he wants to delete everything and regenerated

Reply:
@yevgenypats I agree, but this will lead to a lot of duplication since all the plugins will need this functionality, and we don't want them to share common libraries in the monorepo. 

How about we remove the option here, but we add a convenience function for deleting directories to the SDK, maybe in another package?

Reply:
I added a helper function in this commit: https://github.com/cloudquery/plugin-sdk/commit/3ac29627e2bd1b6b28e04fcf0c7ba06ea77fb379

Reply:
I actually meant for us to something like `rm -rf *.md` && `go run main.go doc` so a helper function wont be needed because it highly depends on what is in the directory and specifaclly in our case what is in our website so it can also be something like `rm -rf cloudquery*.md` . I think in cobra there is also no logic around deleting - https://github.com/spf13/cobra/blob/main/doc/md_docs.go

Reply:
Ah I see! Okay yeah, makes sense. I'll remove the delete functionality from this PR and update the scripts to run something like `rm -rf *.md` in a follow-up where we migrate to this implementation.